icicle/primitives/field.cuh

```diff
@@ -85,6 +84,26 @@ public:
     return Field{inv.storages[logn - 1]};
   }
 
+  static constexpr HOST_INLINE unsigned get_omegas_count()
+  {
+    if constexpr (has_member_name<CONFIG>()) {
+      return CONFIG::omegas_count;
+    } else {
+      return 0;
+    }
+  }
+
+  template <typename T>
+  static constexpr bool has_member_name()
+  {
+    // if (std::is_same<unsigned, decltype(T::omegas_count)>) {
+    if (sizeof(T::omegas_count) > 0) {
```

Comment:
minor comment but I would rewrite as 'return sizeof(T::omegas_count) > 0' and avoid the if-else.